languages/go/exercises/concept/maps/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+A friend of you has an old wholesale store called **Gross Store**, the name came from the quantity of the item that the store sell, it's all in [gross unit][gross-unit]. Your friend asked you to implement a point of sale (POS) system for his store, **but first, you want to build a prototype for it, in your prototype, your system will only record the quantity**. Your friend gave you a list of measurements to help you:
+
+| Unit               | Score |
+| ------------------ | ----- |
+| quarter_of_a_dozen | 3     |
+| half_of_a_dozen    | 6     |
+| dozen              | 12    |
+| small_gross        | 120   |
+| gross              | 144   |
+| great_gross        | 1728  |
+
+## 1. Store the unit of measurement in your program
+
+In order to use the measurement, you need to store the measurement in your program.
+
+## 2. Implement `AddItem`
```

Comment:
Could you maybe describe this in terms of what the task aims to achieve, instead of how? The previous task (task 1) is a great example of a good task name. The same  goes for the other tasks.

Reply:
Hmm, this is cumbersome, I do give short description for each step. I prefer the `Implement X` approach for each step, I believe it's easier to digest from leaner perspective, it's easier to refer back what to do when you're going to implement the function, (I know the  wording is a bit different from the other concept exercises), what do you think?

Reply:
I understand your argument, but I don't agree with that. Personally, I found that having some non-technical context helps a lot to "anchor" what I have to do. We didn't have this rule documented, so we just [added this _what_ over _how_ rule to the description of the instructions.md](https://github.com/exercism/v3/pull/1406/files).

Reply:
4fb067d Fixed the tasks heading

Reply:
Great! Could you also update the corresponding headings in the `hints.md` file?